back/engines/commercial/report_builder/app/services/report_builder/queries/methods_used.rb

```diff
@@ -29,6 +29,11 @@ def get_methods_used_in_overlapping_phases(start_date, end_date)
       non_overlapping_phase_ids = Phase.where('end_at <= ? OR start_at >= ?', start_date, end_date).select(:id)
 
       Phase
+        .where(
+          project_id: Project
+            .joins(:admin_publication)
+            .where(admin_publication: { publication_status: %w[published archived] })
+        )
         .where.not(id: non_overlapping_phase_ids)
         .group(:participation_method)
         .count
```

Comment:
```suggestion
Phase
  .joins(:project)
  .merge(Project.not_draft)
  .where.not(id: non_overlapping_phase_ids)
  .group(:participation_method)
  .count
```
I think this is clearer (though it does the same thing).